packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -149,7 +149,10 @@ export class Tooltip extends React.Component<TooltipProps> {
     } = this.props;
     const content = (
       <div
-        className={css(!enableFlip && getModifier(styles, position, styles.modifiers.top), className)}
+        className={css(
+          !enableFlip && (styles.modifiers[position as keyof typeof styles.modifiers] || styles.modifiers.top),
```

Comment:
maybe make this the same as Popover?
`!enableFlip && (styles.modifiers[position as 'top' | 'bottom' | 'left' | 'right'] || styles.modifiers.top)`

Reply:
I like `'top' | 'bottom' | 'left' | 'right'` more than `keyof typeof styles.modifiers`, so I'll try that!